project_proposal.md

```diff
@@ -0,0 +1,229 @@
+## Strings
```

Comment:
Note: You might want to move this file to the "proposals/" directory like it says in the [writeup](https://ucsd-cse231-w21.github.io/proposal/) and rename to something like `proposals/strings.md` since all other proposals will be in that directory.

Reply:
Done :)